backend/src/main/scala/cromwell/backend/Backend.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend
+
+import cromwell.backend.model.{WorkflowDescriptor, JobDescriptor}
+
+/**
+  * Defines basic structure and functionality to work with a backend.
+  */
+trait Backend {
```

Comment:
1st comment: I'm defining the messages this actor will understand and the engine will send. Later if someone wants to create a composition of receive shouldn't be a problem.
2nd: I defined Backend because later if we implement BackendPersistentActor, I will need to implement this.
But... if we don't want to use PA's or EA's in the future I can move these as you describe.